static/js/components/EarthquakePage.jsx

```diff
@@ -97,7 +97,7 @@ DownloadXMLButton.propTypes = {
   }).isRequired,
 };
 
-const sidebarWidth = 190;
+const sidebarWidth = 170;
```

Comment:
Meh, these things really belong in CSS, don't they? Perhaps not possible here.

Reply:
Removing it from all the pages that have it but the sidebar component itself. We don't need it at all anywhere else. I'm removing our current hacky listeners in favor of the cleaner MUI `useMediaQuery` that relies on the more standard breakpoints.